bootstrapvz/common/assets/init.d/expand-root

```diff
@@ -13,8 +13,19 @@
 prog=$(basename $0)
 logger="logger -t $prog"
 
-device_path="/dev/xvda"
+growpart=$(which growpart) || {
+	$logger "growpart was not found on PATH. Unable to expand size."
+	exit 0
+}
 
+root_index="0"
+
+$growpart /dev/xvda $root_index || {
+	$logger "growpart failed. Unable to expand size."
+	exit 0
+}
+
+device_path="/dev/xvda$root_index"
```

Comment:
I think it's better to allow changing the entire device path. This way we can easily switch to `/dev/by-uuid/` or something else if we want to.